website/blog/2020-02-02-upgrade-support.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Introducing Upgrade-Support
+author: Lucas Bento
+authorTitle: React Native Community
+authorURL: https://twitter.com/lbentosilva
+authorImageURL: https://avatars3.githubusercontent.com/u/6207220?s=460&v=4
+authorTwitter: lbentosilva
+category: announcements
+---
+
+After a ton of helpful feedback and discussion by the React Native community, we are eager to announce [Upgrade-Support](https://github.com/react-native-community/upgrade-support), a central place to request and give help when upgrading your app. We welcome every React Native developer to post their issues as well as their solutions to problems encountered when upgrading their applications.
```

Comment:
I think this could use some more context as to why we are doing this.

Here is some additional content that you can choose to use in addition to what you have. 

--------------



After a ton of helpful feedback and discussion by the React Native community, we are eager to announce [Upgrade-Support](https://github.com/react-native-community/upgrade-support), a central place to request and give help when upgrading your app. 

We know that when people upgrade, the occasional XCode build errors, redboxes, or other confusing problems can be frustrating. This requires searching the internet for answers, hopefully stumbling across an issue or blog post where someone had the same error. While we are working on reducing the likelihood you hit these errors with projects like autolinking (LINK NEEDED) and the upgrade helper (LINK NEEDED), we are hoping we can improve the likelihood you can find others solving the same issues.

## How it Works

Upgrade Support is a GitHub repo hosting an issue tracker where people can post any issues they have while upgrading their projects. This should help increase the likelihood that people can find others with the same problems and work together to figure out a solution.

If there is a common problem associated with a part of the upgrade, the Upgrade Helper will begin linking to those issues and solutions helping more people get through upgrades smoothly.